internal/backends/qt/qt_accessible.rs

```diff
@@ -0,0 +1,699 @@
+
+        QAccessibleInterface *focusChild() const override {
+            if (has_focus_delegation) {
+                return const_cast<QAccessibleInterface *>(static_cast<const QAccessibleInterface *>(this));
+            }
+            for (int i = 0; i < childCount(); ++i)  {
+                if (auto focus = child(i)->focusChild()) return focus;
+            }
+            return nullptr;
+        }
+
+        int childCount() const override;
+
+        int indexOfChild(const QAccessibleInterface *child) const override {
+            return m_children.indexOf(child->object()); // FIXME: Theoretically we can have several QAIs per QObject!
+        }
+
+        QAccessibleInterface *child(int index) const override {
+            if (0 <= index && index < m_children.count())
+                return QAccessible::queryAccessibleInterface(m_children[index]);
+            return nullptr;
+        }
+
+        void setText(QAccessible::Text t, const QString &text) override {
+            Q_UNUSED(t); Q_UNUSED(text);
+        }
+
+        QAccessible::Role role() const override {
+            return m_role;
+        }
+
+        QRect rect() const override {
+            auto item = rustItem();
+            QRectF r = rust!(Slint_accessible_item_rect
+                [item: *const ItemWeak as "void*"] -> qttypes::QRectF as "QRectF" {
+                    if let Some(item_rc) = item.as_ref().unwrap().upgrade() {
+                        let geometry = item_rc.borrow().as_ref().geometry();
+
+                        let mapped = item_rc.map_to_window(geometry.origin);
+
+                        qttypes::QRectF {
+                            x: mapped.x as _,
+                            y: mapped.y as _,
+                            width: geometry.width() as _,
+                            height: geometry.height() as _,
+                        }
+                    } else {
+                        Default::default()
+                    }
+                });
+            auto topLeft = mapToGlobal(QPoint(static_cast<int>(r.left()), static_cast<int>(r.top())));
+            auto bottomRight = mapToGlobal(QPoint(static_cast<int>(r.right()), static_cast<int>(r.bottom())));
+            return QRect(topLeft, bottomRight);
+        }
+
+        QAccessibleInterface *childAt(int x, int y) const override {
+            for (int i = 0; i < childCount(); ++i)  {
+                auto c = child(i)->childAt(x, y);
+                if (c) return c;
+            }
+
+            auto r = rect();
+            if (r.contains(x, y)) {
+                return const_cast<QAccessibleInterface *>(static_cast<const QAccessibleInterface *>(this));
+            }
+            return nullptr;
+        }
+
+
+        QColor foregroundColor() const override { return {}; /* FIXME */ }
+        QColor backgroundColor() const override { return {}; /* FIXME */ }
+
+        void virtual_hook(int id, void *data) override { Q_UNUSED(id); Q_UNUSED(data); /* FIXME */ }
```

Comment:
```suggestion
```

let the default implementation be

Reply:
Should we set these? Accessibility is not just about screen readers after all.

But if we do: How do we make use of them?